ts/Core/Renderer/SVG/SVGElement.ts

```diff
@@ -719,29 +810,28 @@ class SVGElement implements SVGElementBase {
      * @function Highcharts.SVGElement#attr
      *
      * @param {string|Highcharts.SVGAttributes} [hash]
-     *        The native and custom SVG attributes.
+     * The native and custom SVG attributes.
      *
      * @param {number|string|Highcharts.SVGPathArray} [val]
-     *        If the type of the first argument is `string`, the second can be a
-     *        value, which will serve as a single attribute setter. If the first
-     *        argument is a string and the second is undefined, the function
-     *        serves as a getter and the current value of the property is
-     *        returned.
+     * If the type of the first argument is `string`, the second can be a value,
+     * which will serve as a single attribute setter. If the first argument is a
+     * string and the second is undefined, the function serves as a getter and
+     * the current value of the property is returned.
      *
      * @param {Function} [complete]
-     *        A callback function to execute after setting the attributes. This
-     *        makes the function compliant and interchangeable with the
-     *        {@link SVGElement#animate} function.
+     * A callback function to execute after setting the attributes. This makes
+     * the function compliant and interchangeable with the
+     * {@link SVGElement#animate} function.
      *
      * @param {boolean} [continueAnimation=true]
-     *        Used internally when `.attr` is called as part of an animation
-     *        step. Otherwise, calling `.attr` for an attribute will stop
-     *        animation for that attribute.
+     * Used internally when `.attr` is called as part of an animation step.
+     * Otherwise, calling `.attr` for an attribute will stop animation for that
+     * attribute.
      *
      * @return {Highcharts.SVGElement}
-     *         If used as a setter, it returns the current
-     *         {@link Highcharts.SVGElement} so the calls can be chained. If
-     *         used as a getter, the current value of the attribute is returned.
+     * If used as a setter, it returns the current {@link Highcharts.SVGElement}
+     * so the calls can be chained. If used as a getter, the current value of
+     * the attribute is returned.
      */
     public attr(
```

Comment:
The only documented overload for the `attr` function doesn't land in es-modules' .d.ts, but it should.

Reply:
NVM, all good, it just landed in [there](https://github.com/highcharts/highcharts/pull/23890/changes/887de7abd36ba8f6e4708ae6ec6d5ead93eaf844), but works and it's all that matters 🎉